examples/operator/101_initial_cluster.yaml

```diff
@@ -146,8 +182,8 @@ stringData:
     CREATE USER 'vt_app'@'localhost';
     GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, RELOAD, PROCESS, FILE,
       REFERENCES, INDEX, ALTER, SHOW DATABASES, CREATE TEMPORARY TABLES,
-      LOCK TABLES, EXECUTE, REPLICATION CLIENT, CREATE VIEW, SHOW VIEW,
-      CREATE ROUTINE, ALTER ROUTINE, CREATE USER, EVENT, TRIGGER
+      LOCK TABLES, EXECUTE, REPLICATION SLAVE, REPLICATION CLIENT, CREATE VIEW,
```

Comment:
Why do we need to add the additional grant? This section was originally copied from init_db.sql which has not changed in the same way.
The two should be kept in sync.

Reply:
Same as above, I just copied what we have on VTop

Reply:
Can you check what is embedded in vtop code?

Reply:
Is that in the example / test, or the version that is embedded in vtop code?

Reply:
@deepthi should we keep the `REPLICATION SLAVE` term in our codebase? Is there an alternative?

Reply:
There is no version of the `init_db.sql` embedded in VTop code. But this extra permission to `REPLICATION SLAVE` to `vt_app` is provided in all the examples and tests in VTop. But on `main`, we do not provide this permission to `vt_app`.

I have been able to track where this change occurred. In the PR https://github.com/vitessio/vitess/pull/10080 we removed this privilege from `vt_app`  but we didn't make the corresponding change in VTop. So I guess, the permissions on main are correct and I'll remove it here and create a PR on VTop too.

Reply:
I have fixed it now.

Reply:
PR on VTop for fixing there too - https://github.com/planetscale/vitess-operator/pull/293